include/seccomp.h.in

```diff
@@ -1,6 +1,8 @@
-/**
+/*
```

Comment:
Using `/**` caused the copyright info to be displayed in Doxygen.  It didn't render nicely, so I switched it back to a regular comment.

Reply:
Thanks for the explanation, I think that's okay; the doxygen docs shouldn't be a replacement for the license and copyright info already present in the source repo.